ui/src/pages/overview/entities/entities-picker.tsx

```diff
@@ -24,7 +25,7 @@ export const EntitiesPicker = ({
         label: e.name,
       }))}
       showClear={false}
-      value={value}
+      value={hasValue ? value : undefined}
```

Comment:
This is because I noticed if no data source is set, backend still sets a value for it. But in FE it looks a bit strange to show "empty string" as selected.

<img width="982" alt="Screenshot 2024-03-18 at 22 53 04" src="https://github.com/RolnickLab/ami-platform/assets/11680517/d08cab6c-874e-4ccc-a2b7-43278906eead">

Reply:
> This is because I noticed if no data source is set, backend still sets a value for it. But in FE it looks a bit strange to show "empty string" as selected.
> 
> <img alt="Screenshot 2024-03-18 at 22 53 04" width="982" src="https://private-user-images.githubusercontent.com/11680517/313850299-d08cab6c-874e-4ccc-a2b7-43278906eead.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTExNDk3NDQsIm5iZiI6MTcxMTE0OTQ0NCwicGF0aCI6Ii8xMTY4MDUxNy8zMTM4NTAyOTktZDA4Y2FiNmMtODc0ZS00Y2NjLWEyYjctNDMyNzg5MDZlZWFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIyVDIzMTcyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1MjdhZTU2OWEzYjM3M2E3NzIzMmQ4ZDQ5NjllYjZlMzI5YzMzZDIxYTFjOGNkNTM0YWVlMjI4NTgwNjczYWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tPIkAA3neMECMNZx91ZUQqpJvRW4XdISgzXW_D25MYk">

I fixed the bogus the data source! merging this PR, thank you.